packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added
+  * Added seleniumServiceUrl parameter for locating selenium service.
+  * Added port parameter for specifying port mapping for the selenium service or the external selenium grid.
```

Comment:
@mjhenkes  Ok with how I've adjusted these items?

Reply:
This looks fine.